### ISSUE TITLE
Maps loading in too slow

### ISSUE THREAD

**Issue by crumpledmint** [ppy/osu#31187]
### Type

Performance

### Bug description

Sometimes it takes like 10 seconds for map to load, idk what it checks or why is it behaving this way but this is unbearable, considering how fast and consistent stable works in that regard

### Version

2024.1208.0-lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/18187861/compressed-logs.zip)

**Comment by peppy** [comment]
To confirm, you mean importing maps right?

**Comment by crumpledmint** [comment]
> To confirm, you mean importing maps right?

No, I mean a map loading for play. It is better with quick retry, certainly, but with pause retry (or just loading the map initially) It sometimes takes an enormous amount of time, which never happened on stable, here is a [clip](https://clips.twitch.tv/OilyRelievedStarFutureMan-LYHlslJNod_QHCzD) of this. Right after this I went to write the issue, so maybe it will show up in the logs

**Comment by peppy** [comment]
Hmm, looks like it might be network related

**Comment by SailorSnoW** [comment]
RARELY happen to me also, but this is a thing yes.
I really think it's network related as when the thing happen, beatmap leaderboards are slower to load too.

**Comment by crumpledmint** [comment]
> Hmm, looks like it might be network related. Are you on slow network or using any kind of VPN / rerouting?

No, I don't use vpn or other rerouting methods. I am [connected](https://www.speedtest.net/result/17152951801) via ethernet and usually there are no internet issues in other applications/games, this also never happened in stable but I think it works a bit differently there because it doesn't check anything beforehand as far as it seems

**Comment by ominoussage** [comment]
Definitely network related. Play the game without internet, beatmaps load noticeably quicker.

**Comment by peppy** [comment]
We're tracking this in many other places, so will close this one.
